apps/cookbook/src/app/showcase/header-showcase/header-showcase.component.html

```diff
@@ -48,6 +48,23 @@ <h2>Flag</h2>
     </cookbook-example-viewer>
   </div>
 
+  <div class="header-example-container">
+    <h2>Flag with custom content</h2>
+    <p>
+      Flag can also be used with custom content. To use custom content you simply apply the
+      directive
+      <code>*kirbyHeaderCustomFlagSection</code>
+      .
```

Comment:
Consider rewording to focus on using 'custom flag' throughout. 
```suggestion
     Custom markup can also be added to the header as a flag. This can be achieved by applying the directive
      <code>*kirbyHeaderCustomFlag</code>
      to your custom flag markup.
```